monitors.py

```diff
@@ -63,6 +65,32 @@ def run(self, env, city: City):
             yield env.timeout(self.f / TICK_MINUTE)
             # self.plot()
 
+class InfectionMonitor(BaseMonitor):
+    def run(self, env, city:City):
+        while True:
+            d = {
+                'store':0,
+                'park':0,
+                'workplace':0,
+                'misc':0,
+                'household':0
+            }
+            for l in d.keys():
+                d[l] = sum([h.is_infectious for h in city.humans if h.location.location_type==l])
+            # self.data = d if len(self.data)==0 else Counter(self.data).update(Counter(d)) 
+            if len(self.data)==0:
+                self.data=d
+            else:
+                d1 = Counter(self.data)
+                d2 = Counter(d)
+                d1.update(d2)
+                self.data = dict(d1)
```

Comment:
Is there an advantage to defining a new `dict`? How about using `defaultdict` to keep a running count? I am just thinking if there is an advantage to the above.

Reply:
this was to handle the non-dict return type on using counter.update.
Changed this in recent commit to use + instead of `update` since it gives the same result and avoids the creation of the temp dicts.